src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/GruppiPGPage.java

```diff
@@ -0,0 +1,417 @@
+package it.pn.frontend.e2e.pages.destinatario.personaGiuridica;
+
+import it.pn.frontend.e2e.common.BasePage;
+import it.pn.frontend.e2e.utility.WebTool;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+public class GruppiPGPage extends BasePage {
+
+    private final AccediAreaRiservataPGPage accediAreaRiservataPGPage = new AccediAreaRiservataPGPage(this.driver);
+    private final Actions actions = new Actions(this.driver);
+
+    @FindBy(id = "side-item-Gruppi")
+    WebElement gruppiButton;
+    @FindBy(id = "name")
+    WebElement inputNomeGruppo;
+    @FindBy(id = "description")
+    WebElement inputDescrizioneGruppo;
+    @FindBy(id = "product-select")
+    WebElement selectSelezioneProdotto;
+    @FindBy(id = "members-select")
+    WebElement selectSelezioneMembri;
+    @FindBy(xpath = "//button[contains(text(), 'Conferma')]")
+    WebElement buttonConferma;
+
+    public GruppiPGPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void clickGruppiButton() {
+        this.js().executeScript("arguments[0].scrollIntoView(true);", gruppiButton);
+        getWebDriverWait(10).withMessage("Il bottone gruppi non è visibile").until(ExpectedConditions.visibilityOf(gruppiButton));
+        gruppiButton.click();
+        WebTool.switchToOtherTab();
+    }
+
+    public void loginGruppi(String nome, String pwd) {
```

Comment:
insert this method in background test

Reply:
Then i can't import backgroundTest in this file because this file is outside test section